### ISSUE TITLE
Instructor audit logs: ensure end date is not before start date

### ISSUE THREAD

**Issue by halfwhole** [TEAMMATES/teammates#11235]
Page: `/web/instructor/logs`

Currently, the date picker for 'search period until' allows you to input a date that is before the date from 'search period from'. It throws an error only after pressing the search button:

<img width="1237" alt="Screenshot 2021-06-29 at 12 23 43 AM" src="https://user-images.githubusercontent.com/41856541/123670911-4c1f7980-d870-11eb-9fbc-2da70c5a2930.png">

Not too urgent as the functionality is not affected. It seems to have a rather easy fix, by changing the min date option for the date picker

**Comment by damithc** [comment]
When fixing this, keep in mind that the user might set start/end dates any number of times, in any order. i.e., while typically a user will set the start first and then the end and then click search, that may not always be the case.

**Comment by rhit-namburs** [comment]
Hi I would like to work on this issue.

**Comment by ypinhsuan** [comment]
@rhit-namburs Hi please go ahead

**Comment by ypinhsuan** [comment]
@rhit-namburs Hi! Sorry I didn't realize there was an open PR for this issue earlier, please look for another issue to work on.

**Comment by sshamouilian** [comment]
it looks like no one is working on this at the moment. Can I take a stab at it? I have to fork a repo and do a PR for a class project by this Friday...

**Comment by t-cheepeng** [comment]
@sshamouilian You are free to work on the PR
